RecoTracker/LSTCore/src/alpaka/Event.h

```diff
@@ -0,0 +1,204 @@
+#ifndef Event_cuh
+#define Event_cuh
+
+#include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
+#include "RecoTracker/LSTCore/interface/alpaka/Module.h"
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+
+#include "Hit.h"
+#include "ModuleMethods.h"
+#include "Segment.h"
+#include "Triplet.h"
+#include "Kernels.h"
+#include "Quintuplet.h"
+#include "MiniDoublet.h"
+#include "PixelTriplet.h"
+#include "TrackCandidate.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/host.h"
+
+namespace SDL {
+  template <typename TAcc>
+  class Event {};
```

Comment:
To me adding a third meaning to the word "event" is confusing (but maybe that's just me).

Reply:
It's at least the two of us.

This looks _a lot_ like something that was developed ignoring common practices, and is now being shoehorned into an existing application.

Reply:
Do you have any suggestions for better name options? All I can think of is something like `LSTRunner`.

Alternative, I think we can try to merge this into the `LST` class. I considered doing this before, but I haven't checked how feasible this is.